pkg/reconciler/v1alpha1/subscription/subscription.go

```diff
@@ -158,6 +150,8 @@ func (r *reconciler) reconcile(subscription *v1alpha1.Subscription) error {
 		return nil
 	}
 
+	var err error
+
 	// Verify that `channel` exists.
 	_, err = r.fetchObjectReference(subscription.Namespace, &subscription.Spec.Channel)
```

Comment:
Rather than `var err error` above, just use `:=` here.

Reply:
Agreed. That's the style we've used in the rest of the code.